sample/build.gradle

```diff
@@ -1,35 +1,33 @@
-apply from: '../dependencies.gradle'
-
-apply plugin: 'com.android.application'
-apply plugin: 'kotlin-android'
-apply plugin: 'kotlin-android-extensions'
+plugins {
```

Comment:
👍, I read in some Gradle docs that this is more efficient, and recommended in the new Gradle versions.